go.yaml

```diff
@@ -15,8 +15,16 @@ package:
       - build-base
       - bash
 
+secfixes:
+  1.19.3-r0:
+    - CVE-2022-41716
+
 environment:
   contents:
+    repositories:
```

Comment:
This isn't needed and shouldn't be present.  Instead, use `gsutil -m rsync` to clone the repository locally.  I can add some helpers to the Makefile to help with this.